classic/docs/reference/_category_.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What is this category for? There's nothing in it.
Can we just delete it for now?